### PR TITLE
Disable stalled PR workflow

### PR DESCRIPTION
**Pull request by kotwanikunal**
### Description
- Disabling stalled PR workflow which is currently leading to PRs being closed without resolutions

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [x] Commits are signed per the DCO using --signoff
- [ ] Commit changes are listed out in CHANGELOG.md file (See: [Changelog](../blob/main/CONTRIBUTING.md#changelog))

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by anasalkouz — APPROVED** [review]

**Review by VachaShah — APPROVED** [review]

**Review by owaiskazi19 — APPROVED** [review]
Do we want to remove this workflow completely or just disable for now?

**Review by dreamer-89 — APPROVED** [review]

File: .github/workflows/stalled.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ jobs:
           stale-pr-label: 'stalled'
           stale-pr-message: 'This PR is stalled because it has been open for 30 days with no activity. Remove stalled label or comment or this will be closed in 7 days.'
           close-pr-message: 'This PR was closed because it has been stalled for 7 days with no activity.'
```
Comment (dreamer-89) [inline .github/workflows/stalled.yml:25]: nit: Do we need to remove numbers 7, 30 or change to X until we decide on right number

**Review by CEHENKLE — APPROVED** [review]

File: .github/workflows/stalled.yml (hunk 1 above)
Comment (kotwanikunal) [reply .github/workflows/stalled.yml]: Deleted the file all together!

**Comment by kotwanikunal** [comment]
#9208 

> ### Gradle Check (Jenkins) Run Completed with:
> * **RESULT:** FAILURE ❌
> * **URL:** https://build.ci.opensearch.org/job/gradle-check/25644/
> * **CommitID:** [a973c3b](https://github.com/opensearch-project/OpenSearch/commit/a973c3bbf367112c72515a4e49d9837b26010360)
>   Please examine the workflow log, locate, and copy-paste the failure(s) below, then iterate to green.
>   Is the failure [a flaky test](https://github.com/opensearch-project/OpenSearch/blob/main/DEVELOPER_GUIDE.md#flaky-tests) unrelated to your change?

**Review by andrross — APPROVED** [review]

**Comment by kotwanikunal** [comment]
#9731 

> ### Gradle Check (Jenkins) Run Completed with:
> * **RESULT:** FAILURE ❌
> * **URL:** https://build.ci.opensearch.org/job/gradle-check/25653/
> * **CommitID:** [a973c3b](https://github.com/opensearch-project/OpenSearch/commit/a973c3bbf367112c72515a4e49d9837b26010360)
>   Please examine the workflow log, locate, and copy-paste the failure(s) below, then iterate to green.
>   Is the failure [a flaky test](https://github.com/opensearch-project/OpenSearch/blob/main/DEVELOPER_GUIDE.md#flaky-tests) unrelated to your change?

**Comment by dblock** [comment]
I think we should replace this workflow with something that says "it's stalled, but is it green? if it's not green, make it green. Otherwise if it's green, tag maintainers to review."

**Pull request merged**